server/models/Column.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const columnSchema = new mongoose.Schema(
+  {
+    name: {
+      type: Array,
+      required: false,
+    },
```

Comment:
Name should be type of string, not array; also, columns should likely be required at all times